### PR TITLE
Add "multi" as a recognized source/ext lang

### PR DESCRIPTION
**Pull request by AntsyLich**
Related to tachiyomiorg/tachiyomi-extensions#12892

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -258,6 +258,7 @@
 
       <!-- Extension section -->
     <string name="all_lang">All</string>
+    <string name="multi_lang">Multiple</string>
```
Comment (vetleledaal) [inline app/src/main/res/values/strings.xml:261]: Multi is already a well-known term in scene releases.

Comment (AntsyLich) [reply app/src/main/res/values/strings.xml:261]: i'll wait for more response

**Comment by arkon** [comment]
What's the difference between all and multi? Should the `all` code just be displayed as "Multi"?

**Comment by AntsyLich** [comment]
it's kinda misleading since they don't contain all the languages so i was thinking it'll more appropriate to change the lang of the exts and sources too

**Comment by alessandrojean** [comment]
Just a remember that changing the language code in the package will change the package and make the app detect the extensions as new ones, so an update will not appear to the users. Maybe then is it an special case that needs to be handled by the app too?

**Comment by vetleledaal** [comment]
I agree with Alessandro, should just change the label so users aren't forced to manually update extensions.

Migrating package names can be considered when the majority of users are on a version that handles changing package names more seamlessly. Hopefully this will also work for extensions migrating to multisrc, maybe rely on each extensions `id`?

**Comment by AntsyLich** [comment]
Hmm makes sense. Updated the pr and also shortened the string to "Multi"

**Pull request merged**